editors/vscode/src/main.ts

```diff
@@ -39,6 +39,26 @@ export async function activate(context: ExtensionContext) {
 	const outputChannel = window.createOutputChannel("Biome");
 	const traceOutputChannel = window.createOutputChannel("Biome Trace");
 
+	const requiresConfiguration = workspace
+		.getConfiguration("biome")
+		.get<boolean>("requireConfiguration");
+
+	// If the extension requires a configuration file to be present, we attempt to
+	// locate it. If a config file cannot be found, we do not go any further.
+	if (requiresConfiguration) {
+		outputChannel.appendLine("Configuration file required, looking for one.");
+		const configFiles = await workspace.findFiles("**/biome.json");
```

Comment:
We should also check for `rome.json` file. Even though it's deprecated, it's still supported.

While doing so, could you add a `TODO` comment like `// TODO: remove in biome 2.0`, please?

Reply:
Yep, I forgot about that one. Thanks for pointing it out.